package.json

```diff
@@ -7,17 +7,20 @@
     "url": "https://github.com/openfisca/openfisca-doc.git"
   },
   "scripts": {
+    "prebuild": "npm run clean",
     "build": "gitbook build",
-    "clean": "rm -rf _book",
+    "clean": "del _book",
+    "deploy": "npm run build && npm run deploy:rsync && git fetch",
```

Comment:
If you only fetch you're not updating your local `gh-pages` branch, so checking it out wouldn't work anyway. If fetching is enough to solve the out-of-date issue, this is linked to your specific setup and I'm not sure that should be part of the scripts.